lib/acts_as_taggable_on_mongoid/models/concerns/tagging_migration.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
```

Comment:
We discussed this.  The comment here does not adequately convey my intention on how this would be used.

I updated the comment and the "migration" name to reflect the original intent better.